documentation/onboard-service.md

```diff
@@ -50,6 +50,12 @@ The code covers most common use cases, but it may be necessary to amend it. Exam
 - The only environment configurations are development and production. The service may need more or use different names.
 - The web application uses `/healthcheck` as health probe. It can be changed to another path or disabled by passing `null`.
 
+### Github actions workflow templates
+Several workflow templates will be created in the /workflows directory.
```

Comment:
Why not new_service/.github directory directly?

Reply:
ok moved to .github/workflows